packages/portal/src/app/shared/errors/not-found/not-found.component.scss

```diff
@@ -0,0 +1,91 @@
+@import url('https://fonts.googleapis.com/css2?family=Black+Ops+One&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Bebas+Neue&family=Black+Ops+One&display=swap');
+
```

Comment:
You've done a great job! Thank you for removing the image file.

I would still recommend we did not import a specific font for a single design element. Font imports tend to be associated with performance issues like FOUC (event tho I noticed you're using display=swap, which is a good practice).

Let's maybe consider using the fonts already in place and then we're ready to go with the merge. Thank you!

Reply:
Okay, noted. ✅
Just pushed the changes 💯